docs/guides/operator-guide-azure-enclave.md

```diff
@@ -24,7 +24,7 @@ UID2 Private Operator for Azure is not supported in these areas: Europe, China.
 
 ## Prerequisites
 
-Before deploying the UID2 Private Operator for Azure, complete these prerequisite steps:
+Before deploying the UID2 <Link href="../ref-info/glossary-uid#gl-private-operator">Private Operator</Link> for Azure, complete these prerequisite steps:
```

Comment:
Can the link we somehow encorporated in the previous intro section instead? Like how it is done for the AWS private operator.

Reply:
Done.